e2e-tests/gef/cypress/integration/clone-gef-deal.spec.js

```diff
@@ -0,0 +1,282 @@
+      cy.url().should('eq', relative(`/gef/application-details/${MIAapplication}/supporting-information/financial-statements`));
+      cy.uploadFile('file1.png', `/gef/application-details/${MIAapplication}/supporting-information/financial-statements/upload`);
+      uploadFiles.uploadSuccess('file1.png');
+    });
+
+    it('should upload files to the `Financial Forecasts` section', () => {
+      uploadFiles.supportingInfoFinancialForecastsButton().click();
+      cy.url().should('eq', relative(`/gef/application-details/${MIAapplication}/supporting-information/financial-forecasts`));
+      cy.uploadFile('file1.png', `/gef/application-details/${MIAapplication}/supporting-information/financial-forecasts/upload`);
+      uploadFiles.uploadSuccess('file1.png');
+
+      cy.uploadFile('file2.png', `/gef/application-details/${MIAapplication}/supporting-information/financial-forecasts/upload`);
+      uploadFiles.uploadSuccess('file2.png');
+    });
+
+    it('should upload files to the `Financial Commentary` section', () => {
+      uploadFiles.supportingInfoFinancialCommentaryButton().click();
+      cy.url().should('eq', relative(`/gef/application-details/${MIAapplication}/supporting-information/financial-commentary`));
+      cy.uploadFile('file2.png', `/gef/application-details/${MIAapplication}/supporting-information/financial-commentary/upload`);
+      uploadFiles.uploadSuccess('file2.png');
+
+      cy.uploadFile('file3.png', `/gef/application-details/${MIAapplication}/supporting-information/financial-commentary/upload`);
+      uploadFiles.uploadSuccess('file3.png');
+    });
+
+    it('should upload files to the `Corporate Structure` section', () => {
+      uploadFiles.supportingInfoCorporateStructureButton().click();
+      cy.url().should('eq', relative(`/gef/application-details/${MIAapplication}/supporting-information/corporate-structure`));
+      cy.uploadFile('file4.png', `/gef/application-details/${MIAapplication}/supporting-information/corporate-structure/upload`);
+      uploadFiles.uploadSuccess('file4.png');
+
+      cy.uploadFile('file5.png', `/gef/application-details/${MIAapplication}/supporting-information/corporate-structure/upload`);
+      uploadFiles.uploadSuccess('file5.png');
+    });
+
+    it('should upload files to the `Debtor Creditor` section', () => {
+      uploadFiles.supportingInfoDebtorCreditorButton().click();
+      cy.url().should('eq', relative(`/gef/application-details/${MIAapplication}/supporting-information/debtor-creditor-reports`));
+      cy.uploadFile('file4.png', `/gef/application-details/${MIAapplication}/supporting-information/debtor-creditor-reports/upload`);
+      uploadFiles.uploadSuccess('file4.png');
+
+      cy.uploadFile('file5.png', `/gef/application-details/${MIAapplication}/supporting-information/debtor-creditor-reports/upload`);
+      uploadFiles.uploadSuccess('file5.png');
+    });
+
+    it('should populate the `Security Details` section', () => {
```

Comment:
Do we need to check any other sections on the main deal page (below the banner) are _not_ populated? i.e Exporter section should be 'incomplete', things like that?

Reply:
the exporter section needs to be cloned over as well - so everything gets copied, including the status of a specific section.

Reply:
OK great makes it easier 👍